Nosenti/certificate-manager---Innocent
# Task8-KAN-39 Additional certificate fields

#### PR description
- This PR adds additional fields to the certificate. You can add participants. 

#### Ticket / Jira Link
- [KAN-39](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3A67b4d316-bd6f-4b16-a625-69afa0078a10&selectedIssue=KAN-39)

#### Checklist:

#### How To Test Changes
- Click on Add Participant button. the popup should show. 
- Filter the participants. Select participants and they should be visible on the page in the tabular form. 
- You should be able to remove a participant after adding them. 


#### Screenshots:
![search_participants](https://github.com/user-attachments/assets/3d921a46-83f5-4eeb-9c77-a6314b2e9485)
![image](https://github.com/user-attachments/assets/d861ff0a-69da-486d-98df-f783b5d160bd)
